### DISCUSSION TITLE
🧩 FEATURE REQUEST:  Bluesky icon

### DISCUSSION THREAD

**Discussion by walterowisk** [dailydotdev/daily#1421] [Feature Requests]
Add Bluesky icon (https://bsky.social) in Profile Social Links section.

![image](https://github.com/user-attachments/assets/0effdab7-2dad-43f3-bb72-8e96ab06d673)

**Comment by idoshamun** [comment]
Hi @walterowisk,

Thanks for the suggestion :)

**Comment by includenull** [comment]
+1 for this, I'm sad I couldn't even put it in the Mastodon field for now 😞

**Comment by mheob** [comment]
+1 for this.

More and more activity in my bubble is moving from X to Bluesky. So I see this as a benefit.

**Comment by idoshamun** [comment]
You can now add your bluesky profile :)

**Reply by walterowisk** [reply]
Yes, I am already using it. Thank you.
I hope you will consider my other Feature Request about adding the Behance profile.
